tests/dbaas_policy.robot

```diff
@@ -7,27 +7,201 @@ Resource            ../resources/keywords/dbaas_policy.resource
     Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
     When User Filters Project ${DBaaSTestNS} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     Then Import Provider Account Failure For DBaaS Policy
 
-Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy
+Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy on Admin View
     [Tags]  dbaas-test-03
     Given The User Logs In To Openshift As OCP_LDAP_DEV
     When User Filters Project ${DBaaSTestNS} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Project Admin To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-04
+    Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
+    When User Navigates To Import MongoDB Provider Account On Developer View
+    Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Service Admin To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-05
+    Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
+    When User Navigates To Import MongoDB Provider Account On Developer View
+    Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-06
+    Given The User Logs In To Openshift As OCP_LDAP_DEV
+    When User Navigates To Add Topology Screen From Developer View      MongoDB     ${DBaaSTestNS}     True
+    Then Cloud Hosted Database Tile Not Loaded Under Add Topology Screen
+
+Scenario: Create DBaaSPolicy As Project Admin From Error screen on Admin View And Imports MongoDB
+    [Tags]  dbaas-test-07
+    Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_PROJECT_ADM
+    Then User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Imports MongoDB
+    [Tags]  dbaas-test-08
+    Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
+    Then User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+Scenario: Create DBaaSPolicy As Developer From Error screen on Admin View
+    [Tags]  dbaas-test-09
+    Given The User Logs In To Openshift As OCP_LDAP_DEV
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_DEV
+    Then DBaaSPolicy Creation Forbidden
```

Comment:
Is it possible to do these developer actions under admin view?

Reply:
Developer persona is restricted to perform some actions and that's what is been tested on line 69 as the application should forbid the user.

Reply:
IMHO, it seems fine for me, but let me know if this needs to be addressed.